NEWS.md

```diff
@@ -14,6 +14,8 @@
 
 1. `by=.EACHI` when `i` is keyed but `on=` different columns than `i`'s key could create an invalidly keyed result, [#4603](https://github.com/Rdatatable/data.table/issues/4603) [#4911](https://github.com/Rdatatable/data.table/issues/4911). Thanks to @myoung3 and @adamaltmejd for reporting, and @ColeMiller1 for the PR. An invalid key is where a `data.table` is marked as sorted by the key columns but the data is not sorted by those columns, leading to incorrect results from subsequent queries.
 
+2. A class could get modified when `j` returns an object whose class is inherited from data.table, a regression in v1.9.4, [#4324](https://github.com/Rdatatable/data.table/issues/4324). Thanks to @HJAllen for reporting and @shrektan for the PR.
```

Comment:
Just the news item here to clarify please. How about:

> When `j` returns an object whose class inherits from `data.table`, e.g. your own class `"myclass"` you have defined, or the class from package `X` whose class is `c("X", "data.table", "data.frame")`, the derived class (`"myclass"` or `"X"` in those examples) is no longer incorrectly dropped from the class of the data.table returned.

Reply:
Sure, your suggestion is clearer.